### DISCUSSION TITLE
Is it possible to adjust the timeout period? I'm certain that the service is running properly.

### DISCUSSION THREAD

**Discussion by Karl-Sparticle** [apache/skywalking#11889] [Q&A] [answered]
Hi,

Is it possible to adjust the timeout period? 

I'm certain that the service is running properly.

![image](https://github.com/apache/skywalking/assets/95554023/0edd6b5f-90ff-4c93-a30f-509d3cdbd166)

![image](https://github.com/apache/skywalking/assets/95554023/e79487aa-c989-481c-a762-f6a9ca32c74d)

Thanks

**Comment by wu-sheng** [comment, marked as answer]
We don't expose this I think.
And 10s timeout should be enough, the query should not take so long.

**Reply by Karl-Sparticle** [reply]
Got it. Thanks
